src/drivers/heater/heater.h

```diff
@@ -163,9 +164,34 @@ class Heater : public ModuleBase<Heater>, public ModuleParams, public px4::Sched
 	 */
 	void update_params(const bool force = false);
 
+	/**
+	 * @brief Enables / configures the heater (either by GPIO or PX4IO)
+	 */
+	void heater_enable();
+
+	/**
+	 * @brief Disnables the heater (either by GPIO or PX4IO)
+	 */
+	void heater_disable();
+
+	/**
+	 * @brief Turns the heater on (either by GPIO or PX4IO)
+	 */
+	void heater_on();
+
+	/**
+	 * @brief Turns the heater off (either by GPIO or PX4IO)
+	 */
+	void heater_off();
+
 	/** Work queue struct for the RTOS scheduler. */
 	static struct work_s _work;
 
+	/** File descriptor for PX4IO for heater ioctl's */
+#if defined(HEATER_PX4IO)
```

Comment:
@JacobCrabill HEATER_PX4IO will be defined in heater.cpp which includes heater.h before it could be defined.